tests/functional/modules/test_zos_mount_func.py

```diff
@@ -0,0 +1,330 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) IBM Corporation 2020
+# Apache License, Version 2.0 (see https://opensource.org/licenses/Apache-2.0)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+import tempfile
+
+from ansible_collections.ibm.ibm_zos_core.plugins.module_utils import (
+    data_set,
+)
+
+from ansible_collections.ibm.ibm_zos_core.plugins.module_utils.import_handler import (
+    MissingZOAUImport,
+)
+
+try:
+    from zoautil_py import Datasets
+except Exception:
+    Datasets = MissingZOAUImport()
+
+
+INITIAL_PRM_MEMBER = """/* Initial file to look like BPXPRM */
+/* some settings at the top */
+something = something else
+anotherthing = something I forgot
+
+/* This mount shouldn't change */
+MOUNT FILESYSTEM('IMSTESTU.ZZZ.KID.GA.ZFS')
+      MOUNTPOINT('/python3')
+      TYPE('zFS')
+      MODE(RDWR)
+      SETUID
+      WAIT
+      SECURITY
+      AUTOMOVE
+"""
+
```

Comment:
Missing a state=remount test if its possible, i suspect its not here because there is really no way to validate it remounts , df stats don't share time. Maybe so, it would just be a sanity test then to ensure the path is functional.

Reply:
I didn't set a remount test because I couldn't find a way to distinguish between a no-op and a remount, since the result would be the same.

Reply:
I've copied the double_mount test to do a remount test (mount, remount, check for errors)